lib/appsignal/config.ex

```diff
@@ -162,11 +162,12 @@ defmodule Appsignal.Config do
   end
 
   def request_headers do
-    Application.fetch_env!(:appsignal, :config)[:request_headers] || []
+    Application.get_env(:appsignal, :config, %{request_headers: []})[:request_headers] || []
   end
 
   def ca_file_path do
-    Application.fetch_env!(:appsignal, :config)[:ca_file_path]
+    config = Application.get_env(:appsignal, :config, %{ca_file_path: default_ca_file_path()})
```

Comment:
Would it be possible to add `default_ca_file_path()` as the default value for `:ca_file_path` in `@default_config`, and fetch it from there?

Reply:
We’ve been doing it this way since https://github.com/appsignal/appsignal-elixir/commit/13b823174, which introduces “config sources”. Turns out the runtime configuration there has things we can’t hardcode into the compile-time config.